scripts/settle.py

```diff
@@ -25,23 +25,23 @@ def get_oracle() -> str:
     return data['addresses']['marketOracle']
 
 
-def load_market(address: str):
-    with open('./artifacts/contracts/Market.sol/Market.json') as f:
-        data = json.load(f)
-        abi = data['abi']
-        contract = web3.eth.contract(address=address, abi=abi)
-        return contract
+def load_market(address: str, token: str):
+    response = requests.get(f'https://raw.githubusercontent.com/horse-link/contracts.horse.link/main/deployments/goerli/{token}Market.json')
+    data = response.json()
+    abi = data['abi']
+    contract = web3.eth.contract(address=address, abi=abi)
+    return contract
 
 
 def load_oracle():
 
     address = get_oracle()
 
-    with open('./artifacts/contracts/MarketOracle.sol/MarketOracle.json') as f:
-        data = json.load(f)
-        abi = data['abi']
-        contract = web3.eth.contract(address=address, abi=abi)
-        return contract
+    response = requests.get('https://raw.githubusercontent.com/horse-link/contracts.horse.link/main/deployments/goerli/MarketOracle.json')
```

Comment:
not sure what the pattern here is exactly, but the `load_market` function has an `f` followed by the url (see line 29), just wondering if that should be used here?